cli/rm.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// RmCommand is used to implement 'rm' command.
+type RmCommand struct {
+	baseCommand
+	force bool
+}
+
+// Init initializes RmCommand command.
+func (r *RmCommand) Init(c *Cli) {
+	r.cli = c
+	r.cmd = &cobra.Command{
+		Use:   "rm container",
+		Short: "remove a container",
```

Comment:
I think this command supports removing multiple containers. Could we change Short message to `Remove one or more containers` to be more precise?